string status
int32[2] tasks
bool[2] player1_tasks
bool[2] player2_tasks
int32 go
bool sim_closed
bool player1_topview
bool player2_topview
